packages/expo-sqlite/src/SQLite.ts

```diff
@@ -81,16 +90,28 @@ function addExecMethod(db: any): WebSQLDatabase {
 }
 
 export function openDatabase(
-  name: string,
+  fileInfo: SQLiteFileInfo | string,
```

Comment:
Could you also add support for this to `SQLite.web.ts` so it strips `key` from the `fileInfo` and uses only `name` if an object is provided?

Reply:
I have made the change, but would appreciate if you could double-check `SQLite.web.ts` to see if I did what you were expecting